packages/jest-editor-support/src/TestReconciler.js

```diff
@@ -108,15 +108,24 @@ module.exports = class TestReconciler {
 
   // Do everything we can to try make a one-liner from the error report
   sanitizeShortErrorMessage(string: string): string {
+    if (string.includes("does not match stored snapshot")) {
+      return "Snapshot has changed"
+    }
+
+    if (string.includes("New snapshot was not written")) {
+      return "New snapshot is ready to write"
+    }
+
     return string
       .split('\n')
       .splice(2)
       .join('')
-      .replace('  ', ' ')
-      .replace('Received:', ' Received:')
-      .replace('Difference:', ' Difference:');
+      .replace(/\s\s+/g, ' ')
+      .replace('Received:', ', Received:')
+      .split('Difference:')[0];
```

Comment:
Turned out the two replaces above had some invisible control characters too:

<img width="615" alt="screen shot 2017-05-19 at 15 58 49" src="https://cloud.githubusercontent.com/assets/49038/26253905/c1ef4a1e-3cac-11e7-8507-f5c45b463536.png">